### DISCUSSION TITLE
Moving a tab to new windows doesn't remove it from the tree view

### DISCUSSION THREAD

**Discussion by yhcharles** [Able-Bear-Software/tab-nodes-tree#92] [Bug Report]
What I want to do: drag and move a tab to a new window

Problems:
1. I can't do it in the sidebar of "tab-node-tree", drag a tab only moves it around in current window, it's not possible to move it to new window. Is it possible to support this?
2. then I have to do it form the Chrome tab bar, but when I move a tab out from current window, it's still shown in the sidebar "tab-node-tree" view. This should be a bug that the status in the extension is not updated.

**Comment by Tab-Nodes-Tree** [comment]
@yhcharles 

> What I want to do: drag and move a tab to a new window
> 
> Problems:
> 
> 1. I can't do it in the sidebar of "tab-node-tree", drag a tab only moves it around in current window, it's not possible to move it to new window. Is it possible to support this?

Unfortunately, this is not currently supported by Chrome.

> 2. then I have to do it form the Chrome tab bar, but when I move a tab out from current window, it's still shown in the sidebar "tab-node-tree" view. This should be a bug that the status in the extension is not updated.

I’ve raised a bug ticket (#93) for this issue — thank you for reporting it!

Alternative Solution:
You can use the context menu to move tabs to new windows. While this may not be ideal, it is a functional workaround for the time being.
![image](https://github.com/user-attachments/assets/12f9c8a5-cc69-4841-a00e-4dd8f68336a9)
